### DISCUSSION TITLE
Trying to setup Miryoku ColemakDHK for my Cheapino

### DISCUSSION THREAD

**Discussion by jcs021102** [manna-harbour/miryoku#352] [Q&A] [answered]
This is admittedly my first build, and Ive been attempting to install Miryoku ColemakDHK  for like 4 hours now.

I really tried setting it up without help, but no matter what I do, I just keep getting QWERTY. I've tried flashing it, using the actions installer and then converting from .hex to .uf2 (as im using a RP2040-Zero).

Could someone kindly guide me please, as maybe im just being a moron.

**Comment by manna-harbour** [comment, marked as answer]
In your actions runs you are building for `crkbd/rev1`, not for cheapino.

For cheapino firmware see https://github.com/tompi/cheapino/blob/master/doc/firmware.md. You could add the keyboard definition to your fork of Miryoku QMK, or to a fork of QMK using https://github.com/manna-harbour/miryoku/discussions/287, or fork and build from https://github.com/tompi/cheapino/blob/master/doc/firmware.md#build-your-own-miryoku.
